### ISSUE TITLE
codepen/jsfiddel demo?

### ISSUE THREAD

**Issue by mathiasleroy** [pulsardev/vue-tour#83]
Hello,
I find it a bit hard to get started. 
**I'm not using compiled vue-js** so I don't know how to add vue-tour from your examples in the readme.

A working codepen/jsfiddel would be nice example we could replicate.

Best,

**Comment by mmorainville** [comment]
Hi @mathiasleroy,

It's a good idea but we feel like it's more a work for the community.
All in all, all you have to do to get started is regeristering the plugin on your Vue instance:

```js
Vue.use(VueTour)
```

And then using the component in your template as described in the README.

If anyone want to do (or has already made) a CodePen for it, he's more than welcome!

**Comment by mathiasleroy** [comment]
Thanks! I got it working after a few trying around § days ago.  
I should have done the codepen myself – sorry for not contributing :s – but in the meantime I have found shepherd.js to be more suited to my needs and deleted my vue-tour code.

For anyone looking into this: It didn't require the `Vue.use(VueTour)`, just insert the .js files.  
Then start the plugin from the main app `mounted:` section.
